src/main/java/org/jabref/gui/FXDialogService.java

```diff
@@ -64,6 +64,7 @@ private static FXDialog createDialog(AlertType type, String title, String conten
         alert.setHeaderText(null);
         alert.setContentText(content);
         alert.getDialogPane().setMinHeight(Region.USE_PREF_SIZE);
+        alert.showAndWait();
```

Comment:
This is not a good idea to add it here as is the internal method for creating a dialog. Normally you call one of the public method from the Dialog Service, get a dialog object back and then call show and wait on that object. Just look in other places how it's done